addons/mail/static/src/core/common/message.xml

```diff
@@ -90,6 +90,7 @@
                                                         <div class="overflow-x-auto" t-if="message.message_type and message.message_type.includes('email')" t-ref="shadowBody"/>
                                                         <t t-elif="state.showTranslation" t-out="message.translationValue"/>
                                                         <t t-elif="message.body" t-out="props.messageSearch?.highlight(message.body) ?? message.body"/>
+                                                        <em t-if="message.edited" class="o-mail-edit-label smaller fw-bold text-500"> (edited)</em>
```

Comment:
This currently goes on a new line. Can't we add it on the last line of the message whenever possible?
Or in the message header instead?
Maybe to check with @alexkuhn

Reply:
TBH this is quite minor, we could improve later if ensuring this is inline and looks good takes time ;)